IMOS_ATF-ACOUSTIC/index_tmp.sql

```diff
@@ -0,0 +1,64 @@
+-- insert what has been harvested into the database
+-- creates if needed the table detection_file_index
+-- then creates the views to be used for WMS
+
+-- create the full index table when doesn't exist (= the first time it runs)
+-- should ONLY run ONCE
+-- could disable and safe keep this script here for reference once run
+--/*
+CREATE TABLE IF NOT EXISTS animal_tracking_acoustic_qc.detection_file_index
+(
+    id bigserial PRIMARY KEY,
+    tag_deployment_id varchar(55) UNIQUE NOT NULL,
+    transmitter_id varchar(55),
+    species varchar(55),
+    tag_project varchar(255),
+    time_coverage_start timestamp with time zone,
+    time_coverage_end timestamp with time zone,
+    min_lat double precision,
+    max_lat double precision,
+    min_lon double precision,
+    max_lon double precision,
+    geom geometry(Geometry,4326),
+    first_indexed timestamp with time zone,
+    last_indexed timestamp with time zone DEFAULT Now(),
+    url varchar(255),
+    size integer,
+    FOREIGN KEY (tag_deployment_id)
+        REFERENCES animal_tracking_acoustic_qc.index_tmp(tag_deployment_id)
```

Comment:
It doesn't make sense to me, because it sort of implies you can only have `tag_deployment_id`s in the `detection_file_index` table that also exist in the `index_tmp` table.
I don't think you actually need a foreign key in this case.